packages/terra-navigation-side-menu/src/NavigationSideMenu.jsx

```diff
@@ -396,7 +396,7 @@ class NavigationSideMenu extends Component {
           aria-relevant="additions text"
           refCallback={this.setVisuallyHiddenComponent}
         />
-        <ContentContainer {...customProps} fill className={sideMenuContentContainerClassNames}>
```

Comment:
why was the `fill` removed here and width and height 100% added through css class ?

Reply:
fill is removed here from the content container prop because it has position absolute css styles along with min height of 50 px in the parent container. 

width and height can be removed. will update this.

![Screenshot 2024-05-06 at 5 08 11 PM](https://github.com/cerner/terra-framework/assets/132886427/3b5e2988-6f6e-4bac-97c3-e8ce4d57e68a)